projects/components/src/table/table.component.ts

```diff
@@ -141,7 +141,11 @@ import {
     </div>
 
     <!-- Pagination -->
-    <div class="pagination-controls" *ngIf="this.pageable">
+    <div
+      class="pagination-controls"
+      *ngIf="this.pageable"
+      [style.position]="this.display === 'full-page' ? 'fixed' : 'sticky'"
```

Comment:
Would prefer to use the enum for full page

Reply:
@skjindal93 done.